packages/google_mlkit_face_mesh_detection/android/build.gradle

```diff
@@ -40,5 +40,5 @@ android {
 }
 
 dependencies {
-    implementation 'com.google.mlkit:face-mesh-detection:16.0.0-beta1'
+    implementation 'com.google.mlkit:face-mesh-detection:16.0.0-beta2'
```

Comment:
Face mesh is still on beta1  
https://developers.google.com/ml-kit/vision/face-mesh-detection/android#before_you_begin

Reply:
The website is not updated with the latest.

Here are the latest: https://mvnrepository.com/artifact/com.google.mlkit/face-mesh-detection